tests/bdd/servicing/service_test.go

```diff
@@ -1,13 +1,41 @@
 package servicing
 
 import (
+	"fmt"
+	"github.com/h2non/gock"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/pokt-network/pocket-core/types"
 	"github.com/pokt-network/pocket-core/x/service"
 )
 
 var _ = Describe("Service", func() {
-	PDescribe("Service Configuration", func() {})
+	Describe("Service Configuration", func() {
+		// Contextualize this block on connection type (e.g. HTTP)
+		Describe("Configuring a third party blockchain endpoint list", func() {
+
+			Describe("Parsing a json file", func() {
+
+				Context("Able to unmarshal the blockchain list into chain objects", func() {
+
+					It("should return a nil error", func() {
+						Expect(types.HostedChainsFromFile(chainsfile)).To(BeNil())
+					})
+
+					It("should have created a globally accessible list of blockchains", func() {
+						Expect(types.GetHostedChains().Len()).ToNot(BeZero())
```

Comment:
Shouldn't this test check against the fixture data to make sure the `HostedChain` instances are properly created?

Reply:
I mocked hosted chains using a fixture document that includes correct inputs. Are you suggesting to test the fixture code, cause it has little to do with the servicing functionality?

Reply:
No, what I mean is the design of the test itself, by literally reading the test I get the sense that I could have a list of empty structs and just because the list is full it doesn't mean the list is valid, you see what I'm saying? Behaviour wise we want tests to reflect the expected behaviour literally.

Reply:
Done